src/FastBoard.cpp

```diff
@@ -455,15 +493,8 @@ float FastBoard::final_mc_score(float komi) {
 void FastBoard::display_board(int lastmove) {
     int boardsize = get_boardsize();
 
-    myprintf("\n   ");
-    for (int i = 0; i < boardsize; i++) {
-        if (i < 25) {
-            myprintf("%c ", (('a' + i < 'i') ? 'a' + i : 'a' + i + 1));
-        } else {
-            myprintf("%c ", (('A' + (i-25) < 'I') ? 'A' + (i-25) : 'A' + (i-25) + 1));
-        }
-    }
-    myprintf("\n");
+	myprintf("\n");
```

Comment:
nit: incorrect spacing

This problem is repeated all over. I think your editor is misconfigured. Also if you can, try avoid lines longer than 80 characters.

Reply:
Yes. I did not have VS configured correctly so it was mixing in tabs. I think I have it configured correctly now. All tabs replaced with spaces and I turned on visible white-space to avoid future occurrences. I will try to keep lengths below 80, but that seems pretty low, and there are many lines that exceed that in the original.